experimental/web/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title>Oak emulator</title>
+</head>
+
+<body>
+    <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
```

Comment:
Interesting that vue recommends this as the default way to import it, I wonder what happens when they release a breaking version change. 

Personally I'd recommend to import it as a module, which is versioned & avoids polluting the global namespace.

This would go into the module tag below:

```
import Vue from 'https://cdn.jsdelivr.net/npm/vue@2.6.12/dist/vue.esm.browser.js'
```

via https://vuejs.org/v2/guide/installation.html#CDN

Reply:
Done, thanks!